src/NonCopyable/NonCopyable/NonCopyableAnalyzer.cs

```diff
@@ -101,18 +101,6 @@ public override void Initialize(AnalysisContext context)
                     }
                 }, OperationKind.ArrayInitializer);
 
-                csc.RegisterOperationAction(oc =>
-                {
-                    var op = (ICollectionElementInitializerOperation)oc.Operation;
-
-                    if (!HasNonCopyableParameter(op.AddMethod)) return;
-
-                    foreach (var a in op.Arguments)
-                    {
-                        CheckCopyability(oc, a, InitializerRule);
-                    }
-                }, OperationKind.CollectionElementInitializer);
-
```

Comment:
Is this change related to supporting C# 9?

Reply:
Yeah, it is in the comment about this PR:

> `CollectionElementInitializer` action was removed because the new version of Roslyn for C# 9 treats collection initializers as `Add` method invocations, so a separate handler is not needed.